### PR TITLE
Adding doc to function extractElectrodePointclouds

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Input and Output descriptions are to be determined and reviewed.

### CODE REVIEW

File: src/Functions/extractElectrodePointclouds.m [hunk 1]
```diff
@@ -1,12 +1,29 @@
-%% extractElectrodePointclouds - Preprocess CT Data for Electrode Artifacts
+function [elecsPointcloudStruct, brainMask] = extractElectrodePointclouds(niiCT, varargin)
+% extractElectrodePointclouds - Preprocess CT Data for Electrode Artifacts
+%
+% USAGE:
+%
+%   [elecsPointcloudStruct, brainMask] = extractElectrodePointclouds(niiCT, varargin)
+%
+% INPUTS: 
+%    niiCT:         NiftiMod object of a post-operative CT image
```
Comment (Daniel-Tojal) [inline src/Functions/extractElectrodePointclouds.m:9]: @adhusch Could you please provide the suitable descriptions of the inputs and outputs, and verify if the varargin arguments are listed properly?

**Review by adhusch — APPROVED** [review]

File: src/Functions/extractElectrodePointclouds.m (hunk 1 above)
Comment (adhusch) [reply src/Functions/extractElectrodePointclouds.m:9]: medtronicXMLPlan: Path to XML file defining a Medtronic Stealth Station S7 trajectory plan
methalThreshold: CT metal threshold value

**Review by adhusch — CHANGES_REQUESTED** [review]
medtronicXMLPlan: Path to XML file defining a Medtronic Stealth Station S7 trajectory plan
methalThreshold: CT metal threshold valu

**Comment by laurentheirendt** [comment]
@Daniel-Tojal 👍

**Comment by laurentheirendt** [comment]
Thanks @Daniel-Tojal 👍

**Pull request merged**